thorlcr/activities/thactivityutil.cpp

```diff
@@ -783,9 +783,42 @@ IFileIO *createMultipleWrite(CActivityBase *activity, IPartDescriptor &partDesc,
     Owned<IFileIO> fileio;
     if (compress)
     {
+        // rowdif used if recordSize > 0, fallback set to LZ4
+        unsigned __int64 compMethod = COMPRESS_METHOD_LZ4;
+        // fallback override
+        StringBuffer compType;
+        activity->getOpt(THOROPT_FBACK_COMPRESS_TYPE, compType);
```

Comment:
Should we change the global default from row diff to LZ4 everywhere ??
I will run some perf tests to see the effects.
I do see some good speedups using LZ4 now instead of lzw for spills.